TelenorNBIoT.h

```diff
@@ -97,17 +97,25 @@ class TelenorNBIoT
     bool createSocket(const uint16_t listenPort = 1234);
 
     /**
-     * Receive data.
+     * Receive bytes. If there's no data available it will return 0.
```

Comment:
Document the other error codes `< 0`.  And add `#define`s or `const`s for them.

Reply:
Document how to use this function and its siblings.

Reply:
Updated. Is the documentation ok?

Reply:
Looks good!